cheddar/cheddar-integration-aws/build.gradle

```diff
@@ -24,6 +24,7 @@ dependencies {
     compile "joda-time:joda-time:${jodaTimeVersion}"
     compile "com.fasterxml.jackson.datatype:jackson-datatype-joda:${jacksonVersion}"
     compile 'net.spy:spymemcached:2.11.4' // Used to connect to memcache nodes - the aws version of this file is not in a public repo
+    compile "com.google.code.gson:gson:2.8.0"
```

Comment:
The reason for using Gson was because I found it has some support for excluding the mapping for some fields. Maybe some thinking would be needed, but I think it's not OK to log everything (including sensitive details like password). As a first step I added a filter on the field name "password". What is your thinking about it?